lib/Target/CBackend/CBackend.cpp

```diff
@@ -2858,7 +2863,7 @@ void CWriter::generateHeader(Module &M) {
             break;
           default:
 #ifndef NDEBUG
-            errs() << "Invalid operator type!" << opcode << "\n";
+            errs() << "Invalid operator type 1! " << opcode << "\n";
 #endif
```

Comment:
Added the #! to know where the error occurred. Maybe __LINE__ would be better.

Reply:
yeah, __LINE__ seems easier to keep